assets/agenda/components/AgendaListItem.tsx

```diff
@@ -304,10 +304,10 @@ class AgendaListItem extends React.Component<IProps> {
             >
                 <div className={classes.wrap} tabIndex={0}>
                     <div className={classes.article} key="article">
-                        <LabelGroup>
+                        <Spacer gap='4' noGrow>
```

Comment:
I would probably still like better that `LabelGroup` with `Spacer` inside, I don't like that duplicated code `gap='4' noGrow` everywhere

Reply:
but what will that component do? GroupLabel?

Reply:
it's about code quality, functionality wise it's the same

Reply:
so if we would like to have there `gap='5'` we won't have to look all over the place, but you just open LabelGroup component and change it there

Reply:
so u want to have LabelGroup component back with Spacer inside.. and just copy/paste props from spacer right?

Reply:
I think the LabelGroup should be without props pretty much, just doing:
```ts
const LabelGroup = ({children}) => (
    <Spacer gap='4' nowrap>{children}</Spacer>
);
```

Reply:
Hmm, I'm really not sure about that. We might overdo it sometimes with options, but with props, you can allow only a limited number of options defined by the design. The gaps in Label Group should maybe be 4 and 8, that's it. In some other use cases, such as Cards for instance, it needs to be higher. A generic component does not provide any constraints, so a developer can add `<Spacer gap='16' nowrap>{children}</Spacer>` for labels which we don't want.

Reply:
I'm not saying we shouldn't use the Spacer in other places with different values, it's only for this particular usecase with labels inside

Reply:
This would mean that we wouldn't be able to target that specific value without affecting other places where the spacer is used with the same value. The Label Group here is not a good example; it wouldn't be a big deal if we had just one predefined value for it. However, I'm against that approach as a general solution. Sure, there is a place for such a generic component in many applications, but it is not a silver bullet. There is a need to target just one specific element/component in many scenarios. Using such a generic spacer for everything would make that impossible. We very often have to change the default style based on placement, screen size, device, etc. How would you do that if everything is based on the same CSS? 
Anyway, let's move the conversation elsewhere; I will give you multiple examples where this approach will not work. For this specific case, we are good to go either way.